requirements.txt

```diff
@@ -4,3 +4,4 @@ colorthief==0.2.1
 python-resize-image==1.1.19
 jinja2==2.10.1
 babel==2.7.0
+humanfriendly==8.2
```

Comment:
Allow humanfriendly 8.x.
Actually, if we're using it only for test, it makes no sense to require it here. Maybe it's time we introduce a requirements-dev.txt ?

While you're at it, remove jinja2 and babel that are not used at all and allow for `python-resize-image` 1.1.*